tests/ClientTest.php

```diff
@@ -94,8 +94,7 @@ public function testSelectCollectionInheritsOptions(): void
 
         $this->assertInstanceOf(ReadConcern::class, $debug['readConcern']);
         $this->assertSame(ReadConcern::LOCAL, $debug['readConcern']->getLevel());
```

Comment:
I can go a step further and refactor the `ReadConcern` and `WriteConcern` assertions. Maybe in an other PR.